WatchParty/backend/config/config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  development: {
+    dialect: 'sqlite',
+    storage: './db.development.sqlite',
+    session_secret: 'keyboard cat',
+    salt_rounds: 10,
+    demo_user_password: 'pass1234',
+  },
+  test: {
+    dialect: 'sqlite',
+    storage: ':memory:',
+    session_secret: 'keyboard cat',
+    salt_rounds: 10,
+  },
+  production: {
+    username: process.env.DB_USERNAME,
```

Comment:
Similar here, change to follow for example `username: process.DB_PRODUCTION_USERNAME` if we do not want them to have the same name. Although, since we are checking which env we are in, it may be fine to keep this as is.